hack/e2e.go

```diff
@@ -207,6 +207,11 @@ func run(deploy deployer) error {
 	}
 
 	if *up {
+		// If we tried to bring the Kubemark cluster up, make a courtesy
```

Comment:
How it this connected to Kubemark? I believe it should be:
`If we tried to bring the e2e cluster up`